src/VisualStudio/NuGet.Packaging.VisualStudio/Views/PackageMetadataViewModel.cs

```diff
@@ -84,6 +84,7 @@ public string Title
 			set { storage.SetPropertyValue(value); }
 		}
 
+		[Required]
 		public string PackageVersion
 		{
 			get { return storage.GetPropertyValue<string>(); }
```

Comment:
From this, it looks like the set of the `viewModel.PackageVersion` above isn't needed at all, is it?